contracts/contracts/ProjectRegistry.sol

```diff
@@ -51,6 +51,8 @@ contract ProjectRegistry {
 
     event ProjectCreated(address indexed owner, uint96 projectID);
     event MetadataUpdated(uint96 indexed projectID, MetaPtr metaPtr);
+    event AddedOwner(address owner, uint96 projectID);
+    event RemovedOwner(address owner, uint96 projectID);
```

Comment:
since we use `ProjectCreated`, I was thinking about using `OwnerAdded` and `OwnerRemoved`, wdyt?

Reply:
Good call 😄